src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student6/WorkoutHistoryController6.java

```diff
@@ -0,0 +1,33 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student6;
+
+import com.xpanxion.java.springboot.da1.demo.model.student6.WorkoutHistory6;
+import com.xpanxion.java.springboot.da1.demo.service.student6.WorkoutHistoryService6;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class WorkoutHistoryController6 {
+
+    @Autowired
+    private WorkoutHistoryService6 workoutHistoryService6;
+
+    List<WorkoutHistory6> whIn;
+    List<WorkoutHistory6> whOut;
+
+    @PostMapping("student6/api/v1/member/{memberId}/checkin?time=2021-12-30 13:48:28")
```

Comment:
The time can't be hard coded in the URL.  It should be passed into the `createCheckin()` function, something like this: `@RequestParam @DateTimeFormat(pattern="yyyy-MM-dd H:m:s") Date time`